code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -146,6 +168,7 @@
 		return
 	scrub(us)
 	if(widenet)
+		check_turfs()
```

Comment:
This seems mildly expensive, though I guess it'll be better now

Reply:
oh yeah i forgot air has lower wait than machines. i still dont like having it in two subsystem processes though when its only ever used here so ill add a cooldown

Reply:
done'd